### DISCUSSION TITLE
Temperature doesn't change for calibration tower

### DISCUSSION THREAD

**Discussion by Hipowerguy** [5axes/Calibration-Shapes#61] [Q&A]
Hello,

I have difficulty with the PETG calibration tower.
The temperature doesn't change.  It is always 210C.
I use Cura 4.11 and a Creality CR10Spro.
I am relatively new to 3D printing.
Do I need more than loading the model and slicing it?

Thanks

**Comment by 5axes** [comment]
Yes you have to use a postprocessing script : https://github.com/5axes/Calibration-Shapes/wiki/Customize-script
